config/routes.rb

```diff
@@ -28,6 +28,10 @@
         end
       end
       get 'create_key' => 'products#create_key'
+      get 'new_from_git_remote' => 'products#new_from_git_remote'
+      get 'new_from_rails_template' => 'products#new_from_rails_template'
+      post 'create_from_git_remote' => 'products#create_from_git_remote'
+      post 'create_from_rails_template' => 'products#create_from_rails_template'
```

Comment:
Ah you mean the route helpers. The routes are identical.

Solution for route helpers is to use `as:` http://guides.rubyonrails.org/routing.html#naming-routes

So to get what you had before you would have to do:

```
collection do
  get 'new_from_git_remote', as: 'new_from_git_remote'
  # ... etc
```

Sorry.